src/sqlparser/src/parser.rs

```diff
@@ -4289,6 +4289,26 @@ impl Parser<'_> {
             let value = alt((
                 Keyword::DEFAULT.value(SetTimeZoneValue::Default),
                 Keyword::LOCAL.value(SetTimeZoneValue::Local),
+                preceded(
+                    Keyword::INTERVAL,
+                    cut_err(Self::parse_literal_interval.try_map(|e| match e {
+                        // support a special case for connectors which would send when initializing the connection
```

Comment:
nit: `connectors` -> `clients` to avoid confusion with source / sink

(same wording in e2e)